.github/workflows/trigger-action.yml

```diff
@@ -0,0 +1,24 @@
+name: Trigger UpdateVersions Action in other Repo
+
+on:
+  workflow_dispatch:
+    inputs:
+      app-version: 
+        description: "New App-Version"
+        required: true
+      path-to-chart-yaml:
+        description: "Path to Chart.yaml"
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Call GitHub API to Trigger UpdateVersions Action
+        run: |
+          curl -X POST -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: Bearer ${{ secrets.REPO_2_TOKEN }}" \
+          -H "Content-Type: application/json" \
+          --data '{"event_type": "trigger-event", "client_payload": {"app-version": "${{ inputs.app-version }}", "path-to-chart-yaml": "${{ inputs.path-to-chart-yaml }}"}}' \
+          "https://api.github.com/repos/jan-niklasml/playground/dispatches"
```

Comment:
@jan-niklasml  vielleicht kannst du auch einmal die github cli ausprobieren https://cli.github.com/manual/gh_workflow_run

Reply:
evtl auch noch Sachen in Variablen ausgliedern